dev-docs/source/GettingStartedWithPyCharm.rst

```diff
@@ -0,0 +1,105 @@
+
+2. In the ``Project Interpreter`` sub menu, at the top select the options button and click ``Add...``, a new window should appear titled "Add Python Interpreter". In the menu on the left, select "System Interpreter" (a version of Python with all the correct variables set already exists within Mantid). Click on the ``...`` to open a file browser, and navigate to;
+
+    .. code-block:: sh
+
+        <Mantid Source Directory>/external/src/ThirdParty/lib/python2.7/python.exe
+
+   This is the interpreter, so select "Ok" and apply the changes. This should bring up a list of all the packages associated to the interpreter. There should be many packages, however you should not see PyQt (but instead QtPy).
+
+3. In the ``Project Structure`` sub menu you should see your root directory with the source/build directories both visible (if not, add them). The folder structure should be present in the centre of the window allowing you to mark folders orange (excluded) or blue (source). Source directories will be searched for python code.
+
+   Within the source directory add the following to your sources;
+
+    .. code-block:: sh
+
+        <Mantid Source Directory>/scripts
+        <Mantid Source Directory>/external/src/ThirdParty/lib
+
+   The first folder can be replaced with the folder that contains your code, if you aren't writing code in ``scripts/``. In the Mantid build directory add the following as source folders;
+
+    .. code-block:: sh
+
+        <Mantid Build Directory>/bin/Debug
+
+   here we are setting up PyCharm for the Debug build, you would use ``/bin/Release`` instead if you are building mantid in release mode.
+
+NOTE : In some cases, imports in the code will still be highlighted red when they come from folders within the ``script/`` folder, or from other folders entirely. To fix this simply add the relevant folder that contains the module you are importing in the same fashion as step 3 above. 
+
+Running Files in the Debugger
+-----------------------------
+
+Running python code from within PyCharm which depends on the python API, or PyQt for example requires one extra step. Because the source root labelling from the previous section only affects PyCharm searching and not the run configuration, before running the file we must set up the run configuration correctly.
+
+As an example, create a new file in ``<Mantid Source Directory>/scripts/`` called ``test.py``. Copy into it the Python code below.
+
+4. To edit the configurations go to ``Run->Run...`` and select ``Edit Configurations``. This should open up a sub window. In order to tell PyCharm where to look for python modules and libraries we need to add some folders to the ``PATH`` environment variable. Click on the ``...`` next to the *Environment Variables* box, and hit the ``+`` icon. In the Name column enter "PATH", in the value column enter the following;
```

Comment:
Small detail but I had to click the green plus to make a new configuration before I could see the `environment variables` box.